controllers/membersController.js

```diff
@@ -11,9 +11,15 @@ const getMembers = async (req, res) => {
   try {
     const allMembers = await memberQuery.fetchMembers()
 
+    if (allMembers.length) {
+      return res.json({
```

Comment:
In case if the newer message is required, we can simply return
```js
    return res.json({
      message: allMembers.length ? 'Members returned successfully!' : 'No member found!',
      members: allMembers
    })
```

Reply:
Done. Please check